e2e/tests/upgrades/upgrade_test.go

```diff
@@ -625,11 +625,6 @@ func (s *UpgradeTestSuite) TestV7ToV7_1ChainUpgrade() {
 
 		expected := testvalues.StartingTokenAmount - testvalues.IBCTransferAmount
 		s.Require().Equal(expected, actualBalance)
-
-		// Escrow amount for native denom is not stored in state because pre-upgrade version did not support this feature
```

Comment:
I'm missing some context here, can you explain why we remove this? Trying to navigate through the original PR that added this in order to find a discussion on it was more difficult than I thought :smile:

Reply:
I think it should be added back?

Reply:
Sorry, I should have added a comment in the PR to explain: I removed this because the starting version we are migrating from (i.e v7.0.0) does not support the endpoint to query the total escrow amount. It's only available in the target upgrade version (i.e. v7.1.0).